packages/patterns-react/src/patterns/CardArray/CardArray.js

```diff
@@ -28,46 +30,38 @@ const CardArray = ({ title, content }) => {
   const containerRef = useRef();
 
   useEffect(() => {
-    setCardsHeight();
+    setSameHeight();
+    window.addEventListener('resize', setSameHeight);
```

Comment:
How many times is this getting called in the lifecycle? I know for the footer I had pass in an empty array `[]` to `useEffect()` so it only called it on the first render.